scripts/setup-mapbox-sdk-walkthrough.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Mapbox SDK Credential Setup Script
+# ==================================
+#
+# Purpose:
+# --------
+# This script assists users in setting up the necessary credentials to utilize
+# Mapbox's closed-source SDKs for iOS and Android. It provides step-by-step
+# guidance for obtaining a secret token from Mapbox and subsequently invokes 
+# the "./scripts/configure-mapbox.sh" script to configure the development environment.
+#
+# Background:
+# -----------
+# To use the Mapbox SDKs for iOS and Android development, a secret token 
+# must be obtained from Mapbox's account page. This token is essential for
+# authenticating downloads of the closed-source SDKs during the build process.
+#
+# Usage:
+# ------
+# To configure Mapbox, invoke this script by running the following command from the project's root directory:
+# npm run configure-mapbox
+
+# Color definitions for better readability
+GREEN='\033[0;32m'
```

Comment:
Please use `./scripts/shellUtils.sh` for this instead. Also you can update `success`, `info`, etc... from there to include the `-e` flag for `echo` if you want so that `\n` characters will work as expected 👍🏼

Reply:
ah I didn't know we have utils for bash scripts thank you for the info updated scripts accordingly 🙇